pom.xml

```diff
@@ -90,6 +90,7 @@
         <mapstruct.version>1.6.3</mapstruct.version>
         <maven.version>3.6.3</maven.version>
         <mockito.version>5.18.0</mockito.version>
+        <camunda-bpm-assert.version>7.23.0</camunda-bpm-assert.version>
```

Comment:
Move this to the pom.xml of the java-testing plugin (see comment for dependencyManagement in bom/pom.xml)